### PR TITLE
Give Bird Upward acceleration and limit Movement  Fixes #21

### LINKED ISSUE #21: Give Bird Upward acceleration and limit Movement
To be solved only after #19 and #20 

You may notice that the bird tends to keep going downwards past the canvas. We want to prevent that, for the same we do the following.
(i) In the `update` function of `class Bird`, check the position using the if condition `if(pos.y >=height)`
(ii) Inside the if block specify the exact position such that the ball doesn't go any lower by writing `pos.y=height;`.
(iii) Since the position is constant set velocity to be zero in here itself by writing `vel.mult(0);`

Congratulations on completing the Bird class 🐦 🎉 

To check if a key has been pressed and act accordingly you need to work on `flappy_bird.pde`.
(i) In the `draw` function, write an if statement as  `if (keyPressed)`.
(ii) Declare the upward acceleration as `PVector up = new PVector(0, -5);` inside the if block.
(iii) Update the acceleration using `b.applyForce(up);`.

### PR DESCRIPTION
**Pull request by vridhi-vk**
fixes #21

### CODE REVIEW

File: Game Coders/flappy_bird/flappy_bird.pde [hunk 1]
```diff
@@ -1,13 +1,19 @@
 Bird b;
+
 PVector gravity = new PVector(0, 0.5);
 
 void setup(){
-  size(400,800);
+  size(480,800);
```
Comment (priyanshisharma) [inline Game Coders/flappy_bird/flappy_bird.pde:6]:
```suggestion
  size(400,800);
```
I suppose for this scenario, `size(400,800)` is a standard option, what do you think? is there any particular motivation to change it? 😄

Comment (vridhi-vk) [reply Game Coders/flappy_bird/flappy_bird.pde:6]: Hey sorry I was experimenting with the values!

**Comment by vridhi-vk** [comment]
I hope I have resolved it correctly

File: Game Coders/flappy_bird/flappy_bird.pde (hunk 1 above)
Comment (priyanshisharma) [reply Game Coders/flappy_bird/flappy_bird.pde]: That is really nice, and nothing to be sorry about 🚀

**Comment by priyanshisharma** [comment]
Yes you did 🎉

**Pull request merged**